index.html

```diff
@@ -178,10 +118,170 @@ <h2>Speaker Notes</h2>
           </ul>
         </div>
       </div>
+
+      <div>
+        <h2>Keep it simple</h2>
+        <p>
+          If you don't need speaker notes,
+          don't worry about it.
+          Every child of <code>slide-deck</code>
+          is a slide without any extra markup.
+        </p>
+      </div>
+
+      <div><h2>Keyboard Shortcuts</h2></div>
+      <div>
+        <div slide-canvas>
+          <h3>Always available:</h3>
+          <ul>
+            <li><strong>command-k</strong>: Toggle control panel</li>
+            <li><strong>command-shift-enter</strong>: Start presentation (from first slide)</li>
+            <li><strong>command-enter</strong>: Resume presentation (from active slide)</li>
+            <li>
```

Comment:
This command only worked for me in speaker view.

Reply:
Do you mean the grid view, or the script view or something else? Can you provide the steps to try and reproduce this?

Reply:
Ok, I maybe I was confused by this command `command-enter` being in the `Always Available` list. I wonder if it fits better in the `Active Presentation` list?

This happens in any view after I press `escape` while in a presentation. I understand that ends the presentation, but the last active slide remains highlighted with the dotted outline. This is how the `active` slides look during a presentation, so I thought by using this command I could re-enter the presentation at that same highlighted slide, but it starts the presentation from the beginning.

Reply:
It's hard to know where the bug is there, since I can't reproduce it on my end. Maybe we can screen-share and debug it together?

Reply:
I don't think this is still relevant now that the `esc` shortcut is gone. `command-enter` works as expected now.